packages/paste-design-tokens/package.json

```diff
@@ -18,9 +18,18 @@
     "build:watch": "gulp dev",
     "clean": "rm -rf ./dist",
     "tokens": "gulp default",
-    "test": "jest"
+    "test": "jest",
+    "tsc": "yarn build"
```

Comment:
Since this uses `gulp` to generate both the dist files and the types, I re-use the build command for the `tsc` command rather than trying to split up internals.